src/chat.rs

```diff
@@ -1722,12 +1722,15 @@ pub unsafe fn set_chat_profile_image(
 ) -> Result<(), Error> {
     ensure!(chat_id > DC_CHAT_ID_LAST_SPECIAL, "Invalid chat ID");
 
-    let mut OK_TO_CONTINUE = true;
-    let mut success = false;
-
     let mut chat = Chat::load_from_db(context, chat_id)?;
     let mut msg = dc_msg_new_untyped(context);
-    let mut new_image_rel = None;
+    let new_image_rel;
+
+    macro_rules! returnError {
+        () => {
+            bail!("Failed to set profile image");
+        };
+    }
```

Comment:
why not just call it directly?

Reply:
you're right it just eats up more lines in this case